components/support/migration/src/test/java/mozilla/components/support/migration/FennecMigratorTest.kt

```diff
@@ -419,6 +419,21 @@ class FennecMigratorTest {
         verifyZeroInteractions(accountManager)
     }
 
+    @Test
+    fun `failing migrations are reported - case 9`() = runBlocking {
+        // Fennec SharedPreferences are missing / empty
+        val migrator = FennecMigrator.Builder(testContext, mock())
+            .migrateSettings()
+            .setCoroutineContext(this.coroutineContext)
+            .build()
+
+        with(migrator.migrateAsync().await()) {
+            assertEquals(1, this.size)
+            assertTrue(this.containsKey(Migration.Settings))
+            assertFalse(this.getValue(Migration.Settings).success)
```

Comment:
This is now wrong. Can you assert here that you succeed with `SettingsMigrationResult.Success.NoFennecPrefs`?